docs/source/explore/shop.rst

```diff
@@ -2,35 +2,39 @@
 Bank and Shop
 ############################
 
-If you want to buy items in High Fidelity, head on over to the Marketplace, where you can purchase more than 300 items built by digital artists and creators from around this world. High Fidelity uses their own cryptocurrency, High Fidelity Coin (HFC) to manage your transactions. 
+If you want to buy items in Project Athena, head on over to the Marketplace, where you can purchase more than 300 items built by digital artists and creators from around this world. Project Athena uses their own cryptocurrency, Project Athena Coin (HFC) to manage your transactions. 
 
 .. contents:: On This Page
     :depth: 2
 
 -----------------------------
-Buy High Fidelity Coin
+Buy Project Athena Coin (DISCONTINUED)
```

Comment:
```suggestion
Buy High Fidelity Coin (DISCONTINUED)
```

Reply:
This was very specifically named High Fidelity Coin and was a High Fidelity project. We'll make Athena Coin later. 😜

Reply:
Fixed it.